app/uk/gov/hmrc/eusubsidycompliancestub/models/BusinessEntity.scala

```diff
@@ -19,7 +19,6 @@ package uk.gov.hmrc.eusubsidycompliancestub.models
 import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.eusubsidycompliancestub.models.types.EORI
 
-// TODO consider renaming... UndertakingMember might make more sense
```

Comment:
No action. We've considered and decided that 

1. We don't see an ambiguity or unclarity with the name BusinessEntity
2. Renaming is costly and risky as it would require many changes in all 3 microservices